libs/components/forms/src/lib/modules/selection-box/selection-box-adapter.service.ts

```diff
@@ -131,14 +133,17 @@ export class SkySelectionBoxAdapterService {
       }
     }
 
-    this.renderer.addClass(nativeEl, newClass);
+    this.#renderer.addClass(nativeEl, newClass);
   }
 
   /**
    * Sets the `tabIndex` of the `element` to the provided `tabIndex`.
    */
-  public setTabIndex(element: ElementRef, tabIndex: number): void {
-    const el = element.nativeElement;
-    el.tabIndex = tabIndex;
+  public setTabIndex(element: ElementRef | undefined, tabIndex: number): void {
```

Comment:
This method doesn't make sense if we make the element `undefined`. Also, `tabIndex` is required, which comes after an optional parameter. Possible to revert this change and put the conditional logic on the caller's end?

Reply:
Good call. I think that was from an earlier attempt at things and I missed rolling this one back. I have reverted.